_sass/components/_callouts.scss

```diff
@@ -109,50 +79,23 @@ blockquote.alert-title {
   }
 }
 
-blockquote.alert {
-  margin-left: 0;
-  margin-right: 0;
-
-  > p:first-child {
-    margin-top: 0;
-  }
-
-  > p:last-child {
-    margin-bottom: 0;
-  }
-}
-
-blockquote.alert-title {
-  margin-left: 0;
-  margin-right: 0;
-
-  > p:nth-child(2) {
-    margin-top: 0;
-  }
-
-  > p:last-child {
-    margin-bottom: 0;
-  }
-}
-
-p.example,
-blockquote.example {
+p.tip {
   background: rgba($green-000, 0.1);
   border-left: $border-radius solid $green-300;
   border-radius: $border-radius;
   padding: 0.8rem;
 
   &::before {
     color: $green-300;
```

Comment:
This title is colorized using the same variant as the left border, but the other callout titles aren't colored. I'm thinking we should be consistent there; either colorize them all or none of them. Your thoughts?

Reply:
Ah, good catch.
To be sure the titles have enough color contrast, I made them all the same as the text color, and I forgot the tip callout.